<launch>

  <node name="sense_hands" pkg="sawyer_pong" type = "sense_hands.py"/>
  <node name="pong_MASTER" pkg="sawyer_pong" type="pong_MASTER.py" output="screen">
    <param name = "ball_velocity" value = "0.3"/>
    <param name = "paddle_size" value = "0.1"/>
  </node>
  <node name="armcontrol" pkg="sawyer_pong" type="armcontrol.py"/>

</launch>
